OpenGrok

```diff
@@ -995,8 +995,30 @@ DeployWar()
         FatalError "Missing Deployment Directory ${warTarget}"
     fi
 
+    APP_CONTEXT="source"
+    FINAL_WAR_NAME="source"
+    if [ "$applicationServer" = "Tomcat" ]
+    then
+        if [ ! -z ${OPENGROK_WEBAPP_CONTEXT+x} ]
```

Comment:
what is x ?

Reply:
I think you can do just

```shell
if [ -n "${OPENGROK_WEBAPP_CONTEXT}" ]
```

Reply:
`if [ -n "${OPENGROK_WEBAPP_CONTEXT}" ]` did not work :(

Probably it will be good to add a comment there.

Reply:
Sorry, I noticed that you did not include parameter expansion.

This code check if the variable has zero length, not if it is set.
```bash
if [ -n "${OPENGROK_WEBAPP_CONTEXT}" ]
```

I would like to distinguish two cases:
`OPENGROK_WEBAPP_CONTEXT` is not set -> use `source` context path
`OPENGROK_WEBAPP_CONTEXT` is empty -> use root context path -> `/`

Reply:
Fair enough. I don't like how both the `Configuration` and the shell script contain the `source` default however I cannot think of anything better for now.